packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -214,7 +236,7 @@ export class DevWorkspaceClient extends WorkspaceClient {
     } else {
       sidecarPolicy = SidecarPolicy.MERGE_IMAGE;
```

Comment:
I missed it somehow in the previous PR, but should not theia related attributes be handled by Theia resolved?

Reply:
yes I think we can do a fixup PR on that topic

Reply:
(it's because Vlad is working on enhancement on the lib as well so we'll need to bump the lib anyway)